bodhi/server/scripts/approve_testing.py

```diff
@@ -64,6 +64,13 @@ def main(argv=sys.argv):
                 update.comment(db, text, author=u'bodhi')
                 continue
 
+            # Comment when Disabling Automatic push to stable
+            if update.autokarma and update._composite_karma[1] != 0:
+                print('%s has received negative karma' % update.title)
+                text = unicode(config.get('disable_automatic_push_to_stable'))
+                update.comment(db, text, author=u'bodhi')
+                continue
```

Comment:
I think it would be desirable to make this comment at the moment that the negative karma is disabled, rather than later. This script only runs every 6 hours. I think this would be a good place to add such a comment:

https://github.com/fedora-infra/bodhi/blob/2.4.0/bodhi/server/models/models.py#L1724-L1726